forc/src/cli/commands/plugins.rs

```diff
@@ -18,15 +18,32 @@ pub struct Command {
     describe: bool,
 }
 
+fn get_file_name(path: &Path) -> String {
+    if let Some(Some(path_str)) = path.file_name().map(|path_str| path_str.to_str()) {
+        path_str.to_owned()
+    } else {
+        panic!("Failed to read file name from {path:?}")
```

Comment:
What does the user see when it panics? Curious why we're using `panic!` here instead of `bail!`